src/data/roadmapData.json

```diff
@@ -594,4 +617,4 @@
       ]
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
I think we all have different options set in our code editors - when I save the file, my code editor adds an empty new line at the end of the file (I think this is a safety standard). Some people don't have, and we keep overwriting each other 😄.

It would be cool to settle how the end of the file should look like - maybe some simple eslint could help.

Reply:
true to that. I will introduce some basic config to review